### PR TITLE
Foundriesio test cases

### PR DESCRIPTION
**Pull request by mwasilew**
This patch series contains set of tests used to test Foundries.io Linux microPlatform. It focuses on the OS features - mainly aktualizr-lite daemon. The daemon is used to handle OTA software updates.

### CODE REVIEW

**Review by nareshkamboju — COMMENTED** [review]
Here in the plan,
http://github.com/mwasilew/test-definitions.git

Do you want to keep personal name space "mwasilew" in repo url ?

File: automated/linux/ota-rollback/download-update.sh [hunk 1]
```diff
@@ -7,46 +7,97 @@
...
+        On the unsecured systems it will usually be
+        fw_setenv. On secured systems it might be
+        fiovb_setenv
+	"
+}
+
+while getopts "t:u:s:h" opts; do
+	case "$opts" in
+		t) TYPE="${OPTARG}";;
+        u) UBOOT_VAR_TOOL="${OPTARG}";;
```
Comment (nareshkamboju) [inline automated/linux/ota-rollback/download-update.sh:42]: Indentation.

**Comment by mwasilew** [comment]
@nareshkamboju I guess it's catch-22. Right now plan points to my repository because the tests are there, not in Linaro's repo. I guess it can be corrected in PR. I'll fix it.

File: automated/linux/ota-rollback/download-update.sh (hunk 1 above)
Comment (mwasilew) [reply automated/linux/ota-rollback/download-update.sh:40]: fixed

**Comment by mwasilew** [comment]
@nareshkamboju fixed both comments

File: automated/linux/aklite-download-interrupt/aklite-download-interrupt.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+report_pass "download-pre-received"
+echo "Sleeping 10s"
+
+FOUND=0
+
+while [ ! "${FOUND}" -eq 1 ]
+do
+
+
+    # sleep 5 min
```
Comment (nareshkamboju) [inline automated/linux/aklite-download-interrupt/aklite-download-interrupt.sh:86]: s/sleep 5 min /sleep 1 s/

File: automated/linux/ota-rollback/verify-reboot.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+export UBOOT_VAR_TOOL
+UBOOT_VAR_SET_TOOL=fw_setenv
+export UBOOT_VAR_SET_TOOL
+
+usage() {
+	echo "\
+	Usage: $0
+		     [-type <kernel|uboot>]
+
+	-t <kernel|uboot>
```
Comment (nareshkamboju) [inline automated/linux/ota-rollback/verify-reboot.sh:21]: indentation

File: automated/linux/ota-rollback/verify-reboot.sh [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        Set the name of the tool to set u-boot variables
+        On the unsecured systems it will usually be
+        fw_setenv. On secured systems it might be
+        fiovb_setenv
+	"
+}
+
+while getopts "t:u:s:h" opts; do
+	case "$opts" in
+		t) TYPE="${OPTARG}";;
```
Comment (nareshkamboju) [inline automated/linux/ota-rollback/verify-reboot.sh:41]: indentation

File: automated/linux/ota-rollback/verify-rollback.sh [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+export UBOOT_VAR_TOOL
+UBOOT_VAR_SET_TOOL=fw_setenv
+export UBOOT_VAR_SET_TOOL
+
+usage() {
+	echo "\
+	Usage: $0
+		     [-type <kernel|uboot>]
+
+	-t <kernel|uboot>
```
Comment (nareshkamboju) [inline automated/linux/ota-rollback/verify-rollback.sh:21]: indentation

File: automated/linux/ota-rollback/verify-rollback.sh [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+        Set the name of the tool to set u-boot variables
+        On the unsecured systems it will usually be
+        fw_setenv. On secured systems it might be
+        fiovb_setenv
+	"
+}
+
+while getopts "t:u:s:h" opts; do
+	case "$opts" in
+		t) TYPE="${OPTARG}";;
```
Comment (nareshkamboju) [inline automated/linux/ota-rollback/verify-rollback.sh:41]: indentation

File: automated/linux/ota-rollback/verify-rollback.sh [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+        u) UBOOT_VAR_TOOL="${OPTARG}";;
+        s) UBOOT_VAR_SET_TOOL="${OPTARG}";;
+		h|*) usage ; exit 1 ;;
```
Comment (nareshkamboju) [inline automated/linux/ota-rollback/verify-rollback.sh:44]: indentation

**Review by nareshkamboju — CHANGES_REQUESTED** [review]
Minor comments on indentation and inline comments.

**Review by nareshkamboju — APPROVED** [review]

**Pull request merged**